docs/source/getting_started/install.rst

```diff
@@ -21,7 +21,7 @@ To install PySpark, you can use:
 Python version support
 ----------------------
 
-Officially Python 3.5 and above.
+Officially Python 3.5 to 3.8.
```

Comment:
Okay, seems like this is what we did in `setup.py`.

Reply:
Right. maybe someone just use Python 3.9 and got confused.